docs/API.md

```diff
@@ -0,0 +1,60 @@
+# API
+
+This document covers the documentation for the API the Sinatra service provides.
+
+## Used status codes
+
+Status Code | Description
+------------|--------------
+200			| All good
+400			| Something went wrong
```

Comment:
for 400, something is wrong with the client's request
whereas, 5xx is something went wrong on the server.